python/lsst/cp/pipe/defects.py

```diff
@@ -397,7 +398,6 @@ def _markBlocksInBadColumn(self, x, y, multipleX, defects):
                         limits.append(multipleY[gapIndex])
                         limits.append(multipleY[gapIndex+1])
                     limits.append(maxY)  # maximum last
-                    assert len(limits)%2 == 0, 'limits is even by design, but check anyways'
```

Comment:
Isn't this still the case?  Is this just dropping a check that isn't useful?

Reply:
I mean ... I guess I could bring it back?  It just seems strange to construct something and then assert that it's true.